lib/spack/llnl/util/filesystem.py

```diff
@@ -196,8 +198,7 @@ def _get_mime_type():
     for a specified path
     """
     if sys.platform == "win32":
-        # -h option (no-dereference) does not exist in Windows
-        return file_command("-b", "--mime-type")
+        raise RuntimeError("file not available on Windows")
```

Comment:
I think the error raised in `file_command` is sufficient to cover this and that we do not need to also catch an exception here.

Reply:
Sounds good!